smtlib-backends-process.nix

```diff
@@ -0,0 +1,18 @@
+{ mkDerivation, async, base, bytestring, data-default, lib
```

Comment:
These generated files look all very clean. Would it be worth adding a comment at the top at generation-time, eg.
```
## This file has been generated by `cabal2nix`. Run `nix run .#regen` to update it.
```
(or another name if the name changes as suggested [in this](https://github.com/tweag/smtlib-backends/pull/37/files#r1064468065))